### PR TITLE
rsu: updates for OFS

### PR DESCRIPTION
**Pull request by tswhison**
Change the nextboot option to fpgadefault, with optional parameters
--page and --fallback.  --page allows specifying the primary FPGA boot
image as user1, user2, or factory.  --fallback allows specifying a comma-
separated list of values for the fallback boot sequence.

eg rsu fpgadefault --page=factory --fallback=user2,user1
results in the FPGA boot sequence factory, user2, user1.

Signed-off-by: Tim Whisonant <tim.whisonant@intel.com>

### CODE REVIEW

File: python/opae.admin/opae/admin/tools/rsu.py [hunk 1]
```diff
@@ -47,48 +47,115 @@
...
+        logging.error('--fallback must be accompanied by a --page selection.')
+        raise IOError
+
+    values = ['fpga_' + args.page]
+
+    if args.fallback:
+        fb = args.fallback
+        fb = fb.replace(',', ' ')  # allow a comma-separated list
+        while fb.find('  ') >= 0:
+            fb = fb.replace('  ', ' ') # compress spaces
```
Comment (r-rojo) [inline python/opae.admin/opae/admin/tools/rsu.py:122]: Can this be done with re.sub?
Something like:
```Python
fb = re.sub('\s+',' ', fb)
```

File: python/opae.admin/opae/admin/tools/rsu.py [hunk 2]
```diff
@@ -47,48 +47,115 @@
...
+        fb = fb.strip()
+        fb = fb.rstrip()
+
+        values += ['fpga_' + x for x in fb.split(' ')]
```
Comment (r-rojo) [inline python/opae.admin/opae/admin/tools/rsu.py:126]: This assumes/requires that the fallback sequence values omit 'fpga_'?

**Review by r-rojo — APPROVED** [review]

File: python/opae.admin/opae/admin/tools/rsu.py (hunk 2 above)
Comment (tswhison) [reply python/opae.admin/opae/admin/tools/rsu.py:126]: Yes.  Follows suit with the remainder of the options.

File: python/opae.admin/opae/admin/tools/rsu.py (hunk 1 above)
Comment (tswhison) [reply python/opae.admin/opae/admin/tools/rsu.py]: yep

**Review by anandaravuri — APPROVED** [review]

**Pull request merged**